aplt/scenarios.py

```diff
@@ -98,7 +99,7 @@ def register_forever(reg_delay=30, run_once=0):
     yield connect()
     yield hello(None)
     while True:
-        yield register(random_channel_id())
+        reg = yield register(random_channel_id())
```

Comment:
Yup, this was added in @rpappalax's PR  believe. My local flake found it when I loaded the file. I'm guessing its stricter than the flake8 check we run.